### PR TITLE
Feature/#28 create video

### PR DESCRIPTION
**Pull request by veresspeter**
Tasks:
-
- Create DB connection 
- Create video
  - domain
  - dao
  - service
- Create controller
- Testing
- Update pom.xml-s

Everything works as expected (except the environment for integration tests)

Issues:
-
https://redmine.bsstudio.hu/issues/87
https://redmine.bsstudio.hu/issues/28

Sorry for the nig PR, next time I'll create smaller.

### CODE REVIEW

**Comment by szamat** [comment]
Why didn't you put the VideoService into a specific package?

**Comment by veresspeter** [comment]
Fixed in commit 1aaffb8 in branch feature/#56-archive-video
VideoPortal modul has separate package (but also services can have separate package)

**Comment by szamat** [comment]
I would put that into a package.

File: webcentral-backend-webapp/src/main/java/hu/bme/sch/bss/webcentral/controller/videoportal/VideoController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+public class VideoController {
+
+    private final VideoService videoService;
+
+    public VideoController(final VideoService videoService) {
+        this.videoService = videoService;
+    }
+
+    @PostMapping(produces = "application/json")
+    public final Video createVideo(@Valid @RequestBody final CreateVideoRequest request) {
```
Comment (szamat) [inline webcentral-backend-webapp/src/main/java/hu/bme/sch/bss/webcentral/controller/videoportal/VideoController.java]: I would not return with the Database entity itself, as it expose the structure of database. If you create an explicit response object (which can be the same now as the database entity itself), then the database structure change would not effect directly the contract of you service.

**Comment by veresspeter** [comment]
> I would put that into a package.

Fixed in 151f2da

File: webcentral-backend-webapp/src/main/java/hu/bme/sch/bss/webcentral/controller/videoportal/VideoController.java (hunk 1 above)
Comment (veresspeter) [reply webcentral-backend-webapp/src/main/java/hu/bme/sch/bss/webcentral/controller/videoportal/VideoController.java]: I'll fix this in [#48](https://redmine.bsstudio.hu/issues/48)

**Pull request merged**